wordpress.org/public_html/wp-content/plugins/plugin-directory/shortcodes/class-readme-validator.php

```
@@ -34,7 +34,10 @@ public static function display() {
 		if ( $readme_url && preg_match( '!^https?://([^./]+\.)?wordpress.org/plugins/(?P<slug>[^/]+)!i', $readme_url, $m ) ) {
 			$plugin = Plugin_Directory::get_plugin_post( $m['slug'] );
 
-			if ( $plugin ) {
+			if ( $plugin && (
+				in_array( $plugin->post_status, array( 'publish', 'closed', 'disabled' ), true ) ||
```

PHPStan (2.2.8)

[error] Cannot access property $post_status on WP_Post|true.


(property.nonObject)